frontend/src/views/BookingView.vue

```diff
@@ -130,24 +132,43 @@ const getViewBySlotDistribution = (slots) => {
   return views.invalid;
 };
 
+const handleError = (data) => {
+  errorHeading.value = null;
+  errorBody.value = null;
+
+  if (data?.detail?.id === 'SCHEDULE_NOT_ACTIVE') {
+    errorHeading.value = '';
+    errorBody.value = data.value.detail.message;
+  }
+};
+
 /**
  * Retrieve the appointment from either availability or booking routes.
  * Returns null if there was an error, or the Appointment object if it was successful.
  * @returns {Promise<Object|null>}
  */
 const getAppointment = async () => {
-  const request = call('schedule/public/availability').post({ url: window.location.href.split('#')[0] });
+  let url = null;
+  // Okay we have a slug, lets lookup the actual signature
+  if (route.params.usernameOrSlug && !route.params.signature) {
```

Comment:
If come in with a slug, we look it up in on the server to retrieve their signed url, and then use that to fetch. 

This is a staged approach, and we can easily remove the user's signedUrl, and this intermediate step in the future.